fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/domain/SavingsAccount.java

```diff
@@ -2928,6 +2928,7 @@ public void loadLazyCollections() {
         transactions.size();
         charges.size();
         savingsOfficerHistory.size();
+        if(group != null) { group.getOffice(); }
```

Comment:
Is this left over, or for the lazy loading? I fear future automated code clean up may try to remove this as NOOP... maybe put an inline comment to explain it forces lazy loading, if that's what it is? Or even fake store it in a variable used in a useless expression to prevent NOOP clean up? Or remove if this is just a left over and not actually required.

Reply:
This is for lazy loading. And yes, I agree it makes sense to make this explicit with comment and variable assignment (even though hopefully people will work out what's going on from the function name as well). Will update the PR.